src/transformers/cache_utils.py

```diff
@@ -1870,6 +1870,9 @@ class OffloadedStaticCache(StaticCache):
             The default `dtype` to use when initializing the cache.
         offload_device (`Union[str, torch.device]`, *optional*, defaults to `cpu`):
             The device to offload to. Defaults to CPU.
+        batch_size (`int`, *optional*):
+            The batch size with which the model will be used.
+        layer_device_map (`Dict[int, Union[str, torch.device, int]]`, *optional*): Mapping between the layers and its device.
```

Comment:
```suggestion
        layer_device_map(`Dict[int, Union[str, torch.device, int]]`, `optional`):
            Mapping between the layers and its device. This is required when you are manually initializing the cache and the model is splitted between differents gpus.
            You can know which layers mapped to which device by checking the associated device_map: `model.hf_device_map`.
```
we don't want `batch_size`, but we want to document `layer_device_map`

Reply:
@gante 

if i remove `batch_size` in this pr, https://github.com/huggingface/transformers/blob/main/src/transformers/generation/utils.py#L1636 will raise errors when calling `OffloadedStaticCache`... 

so should i update all caches and https://github.com/huggingface/transformers/blob/main/src/transformers/generation/utils.py#L1636 in our huge PR? :)

Reply:
@gante can you review the last commit: whether the change is what you want?
if yes, i still need to update the failed tests.
thanks

Reply:
and https://github.com/huggingface/transformers/pull/34089/files#diff-357aea2fcf1b51b2a8ad17c7d30044159b1fe1a3f6bd832cced70aca84c8be66, `max_batch_size` is renamed to `batch_size`

correct me if i am wrong: it is better to use `batch_size` to keep consistency...